test/jasmine/tests/lib_date_test.js

```diff
@@ -571,15 +571,19 @@ describe('dates', function() {
                 ],
                 [
                     '%B \'%y WOY:%U DOW:%w',
-                    'August \'12 WOY:32 DOW:1',
+                    'August \'12 WOY:33 DOW:1',
                     'Mesori \'28 WOY:## DOW:##' // world-cals doesn't support U or w
                 ],
+                [
+                    '%B \'%y QOY:%q WOY:%W DOW:%u',
+                    'August \'12 QOY:3 WOY:33 DOW:1',
+                    'Mesori \'28 QOY:3 WOY:48 DOW:1'
+                ],
                 [
                     '%c && %x && .%2f .%f', // %<n>f is our addition
-                    'Mon Aug 13 06:19:34 2012 && 08/13/2012 && .57 .5678',
-                    'Pes Meso 7 06:19:34 1728 && 12/07/1728 && .57 .5678'
+                    '8/13/2012, 6:19:34 AM && 8/13/2012 && .57 .5678',
+                    'Pes Meso 7 6:19:34 AM 1728 && 12/07/1728 && .57 .5678'
```

Comment:
According to https://github.com/d3/d3/blob/master/CHANGES.md#time-formats-d3-time-format
<i>The default U.S. English locale now uses 12-hour time and a more concise representation of the date. This aligns with local convention and is consistent with date.toLocaleString in Chrome, Firefox and Node</i>
```js
var now = new Date;
d3.timeFormat("%c")(new Date); // "6/23/2016, 2:01:33 PM"
d3.timeFormat("%x")(new Date); // "6/23/2016"
d3.timeFormat("%X")(new Date); // "2:01:38 PM"
```